.github/workflows/canister-tests.yml

```diff
@@ -382,9 +382,21 @@ jobs:
           # We are using --partition hash instead of count, because it makes sure that the tests partition is stable across runs
           # even if tests are added or removed. The tradeoff is that the balancing might be slightly worse, but we have enough
           # tests that it should not be a big issue.
+          # mkdir -p /tmp/test-archive/pic/
           ./cargo-nextest nextest run --archive-file canister-tests-${{ matrix.os }}.tar.zst --partition hash:${{ matrix.partition }}
         env:
           RUST_BACKTRACE: 1
+      #     POCKET_IC_LOG_DIR_LEVELS: trace
+      #     POCKET_IC_LOG_DIR: /tmp/test-archive/pic
+      
+      # - name: PocketIC logs
+      #   if: ${{ always() }}
+      #   uses: actions/upload-artifact@v3
+      #   with:
+      #     name: pocket-ic-logs
+      #     path: /tmp/test-archive/pic/*_pocket_ic_*
+
+
```

Comment:
Is this relevant?

Reply:
We added this such that we can gather PocketIC logs when we need them; If this bothers you, the comments can be removed.

Reply:
It bothers me, please remove it :) Or if you really think it's useful leave a note about why this code is commented out.

Reply:
Gotcha. I removed the comments